person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  private :of_age?
+
+  def can_use_services?
+    of_age? || parent_permission == true
+  end
```

Comment:
- [ ] Here both methods are private, try to leave the `private` keyword alone and put the `of_age?` method after, and `can_use_services?` before, kindly implement these changes. 😄
    - Remember that all methods after the `private` keyword will be private methods.